P9Operators/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿/* P9Operators main exercise
+Console.Write("Enter the number of seconds: ");
+int totalSeconds = Convert.ToInt32(Console.ReadLine());
+
+
+double totalDaysFraction = (double)totalSeconds / 86400;
```

Comment:
Instead of these magic numbers, consider using variables:

```csharp
int minuteToSeconds = 60;
int hourToSeconds = 60*minuteToSeconds;
int dayToSeconds = 24*hourToSeconds;

doubleTotalDaysFraction = (double)totalSeconds / dayToSeconds;
```